src/languages/en.js

```diff
@@ -95,7 +95,7 @@ export default {
     hello: 'Hello',
     phoneCountryCode: '1',
     welcomeText: {
-        phrase1: 'With Expensify.cash, chat and payments are the same thing.',
+        phrase1: 'Welcome to new Expensify! Enter your phone or email to continue.',
```

Comment:
This language is confusing. Perhaps `'Welcome to the new Expensify! Enter your phone number or email to continue.'`

@kevinksullivan does it not seem weird to you to have it say `"Welcome to new Expensify!"`?

Reply:
I agree. I think `the New Expensify` is best as the latest mockup shows.